src/ws/session.c

```diff
@@ -59,6 +59,7 @@
 #define DEFAULT_PATH "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 
 static struct passwd *pwd;
+static struct passwd *pwd_buf = NULL;
```

Comment:
Global variables are always NULLed if you don't explicitly initialize them, so this is redundant. But this is fine of course.